README.md

```diff
@@ -1,50 +1,39 @@
+- OpenAI
+- Google
 
-This extension will help as you type to complete the code for you. With the tool, you can have it generate functions and other snippets of code. It will analyze what you have written so far, and it will suggest what it thinks you want. 
+![](ReviewCode.gif)
 
-### Code Discussion
-
-This extension also allows you to discuss or modify the generated code. Instead of trying to find the right prompt to get the code you are looking for, you can suggest changes to the proposed code until you have the code you want. 
-
-## Requirements
-
-The only requirement needed if not running locally would be to set your API key in the setting, which is explained in this document. You will have to get your own API key for the AI service you would like to use like OpenAI or ChatGPT. 
+## Features
 
-<a target="_blank" href="https://code.visualstudio.com/docs/getstarted/settings">How to set your API key</a>
+- Option to review a whole file or just a piece of selected code
+- Formatting suggestions like getting rid of extra lines to make it easier to read
+- Identifying bugs in code
+- Adds Documentation
+- Checks Spelling
+- General improvement suggestions
 
-## Local Set Up
+## Set Up
 
-If you desire to run this extension locally, here are the steps to have it run properly. 
+After installing the byoLAD extension, configure it with your API key from your LLM provider. 
+#### How to set your API key in settings:
+- Go to File and under Preferences select Settings.
+- Once in Settings, click on Extensions.
+- Then select byoLAD under Extensions.
+- Then under APIKey put in your API key from your provider.
+- Then under Model select your LLM provider's API.
 
-#### Specific instructions for setup:
-1. Clone the repo.
-2. Run npm install.
-3. Copy the .vscode.sample folder and rename it to .vscode.
+![](SetAPIKey.gif)
 
-#### To verify that it's working:
-1. With VS code open, press F5.
-2. Another "Extension Development Host" VS code window should open.
-3. With a file open, right click.
-4. Click "Review Code File".
-5. An error message may pop up saying "APIKey not set". If not, then it should have opened a new tab showing your code and the suggested changes.
+> Note: If you want to run the extension locally, go to the [contribute.md](contribute.md)
 
-> Note: The instructions for setting your APIKey in setting is <a target="_blank" href="https://code.visualstudio.com/docs/getstarted/settings">here</a>
+## Who is the BeanLab?
 
-#### Strongly suggested options
-- Install the vscode-eslint and prettier-vscode extensions.
-- Set Prettier as your default formatter in VS code settings.
-- Turn on format on save in VS code settings.
+We are a lab at Brigham Young University, and our goal is to provide useful open-source programs for everyone to use. We are made up of a small group of students and a professor. We enjoy pushing ourselves to make new and improved programs, and we are happy to share them with everyone. 
```

Comment:
Drop the sentence "We are made up...."

Reply:
I think we'll probably replace this whole section with a logo that links to a general Bean Lab page (that I haven't written yet...)

Reply:
Do you have a Bean Lab logo? If so, we could just put it in for now, and then add the link to the image later when you have written the page for the Bean Lab.